woocommerce-gutenberg-products-block.php

```diff
@@ -110,7 +110,20 @@ function() {
 }
 
 add_action( 'plugins_loaded', array( '\Automattic\WooCommerce\Blocks\Package', 'init' ) );
+add_action( 'plugins_loaded', 'woocommerce_blocks_reinitialize_feature_flag', 11 );
 
+/**
+ * Reinitialize the feature flag.
+ *
+ * We want to redefine the lower feature flag to 2 if we're running from the feature plugin.
+ * This file (and code) will only run in the plugin and not WooCommerce core.
+ *
+ * @since 2.7.0
+ */
+function woocommerce_blocks_reinitialize_feature_flag() {
```

Comment:
You could actually set a constant here instead of using plugins_loaded hooks - we know that if this file is loaded we're in **feature plugin context**. So set the constant to 2 and use that later.